Source/WebCore/Modules/mediastream/gstreamer/GStreamerRtpReceiverBackend.cpp

```diff
@@ -94,7 +94,7 @@ RTCRtpParameters GStreamerRtpReceiverBackend::getParameters()
             if (!name.startsWith("extmap-"_s))
                 return true;
 
-            auto extensionId = parseInteger<unsigned short>(name.toStringWithoutCopying().substring(7));
+            auto extensionId = parseInteger<unsigned short>(name.substring(7));
```

Comment:
This can be done without allocating a temporary `String` by using `StringView::substring`. Something like this:
```suggestion
            auto extensionId = parseInteger<unsigned short>(StringView { name }.substring(7));
```

Reply:
name is currently a StringView already.